crates/cards/src/validate.rs

```diff
@@ -1,6 +1,10 @@
-use std::{fmt, ops::Deref, str::FromStr};
+use std::{collections::HashMap, fmt, ops::Deref, str::FromStr};
 
+use common_utils::errors::ValidationError;
+use error_stack::report;
 use masking::{PeekInterface, Strategy, StrongSecret, WithType};
+use regex::bytes::Regex;
```

Comment:
Can we use `regex::Regex` here, or do we necessarily need `regex::bytes::Regex`?

Reply:
We can use `regex::Regex`